### DISCUSSION TITLE
Gout Diet Tips

### DISCUSSION THREAD

**Discussion by KeithCharlieTaylor** [kct2020/goutpal-com-skeleventy#131] [General]
Page URL: https://web.archive.org/web/20220807163458/goutpal.com/1451/low-purine-diet/
[Note: Original page now archived. So the original context no longer applies. But discussing gout diet tips is still valid. Any tips are welcome. Because gout diet is about much more than purine intake]

Page Title: Low Purine Diet

Page Description: 27 Oct 2005 ... A low purine diet can help to reduce gout attacks. GoutPal has tips to help you manage your gout diet. But beware the traps.

## Additional context
~~Most of content is obsolete outside qualified health service providers. Because low purine diet requires professional planning and monitoring. So this page and related content must be summarized correctly for the most interested people.~~

**Comment by hfat** [comment]
My experience with professionals regarding this is that they are ill-informed at best and in any case utterly useless. Obviously, routine tests are worth doing but you'd want to do that anyway regardless of dietary experiments because of the side effects of gout drugs.

**Comment by hfat** [comment]
specific issues with the page include:
-"significant evidence to suggest that vegetables do not cause problems" (we have discussed this many times in the forum and I have never seen such evidence, though I of course agree limiting vegetables isn't going to help most people since there is no reason to think that consumption of *typical amounts of* the problematic vegetables might cause problems)
-"as the body consumes its own purine rich tissue to supply energy" (perhaps simply replace that dubious statements with "in some cases")
-unless I missed that, there is no warning against overeating proteins (a good place to add one would be the weight loss section as that is a popular way of losing weight without starving too badly)

**Comment by kct2020** [comment]
### Tests for Uric Acid Dieters
> Obviously, routine tests are worth doing but you'd want to do that anyway regardless of dietary experiments because of the side effects of gout drugs.

I agree that this seems obvious to me also. But I'm aware of many discussions where people haven't had the basics:
- liver function test
- kidney function test
- uric acid blood test

For anything diet related, I'd also add uric acid excretion rate tests.

My fiendish plan for improving ill-informed professionals is to persuade their patients to educate them. But I accept this might take a while.

### Tips for Uric Acid Dieters
Thanks for commenting on the specific issues. However, my main problem with this page is I need to start again. Because when I wrote it, I had very different views on gout diet than I do now. But, I would like to incorporate your views in whatever I replace it with. Especially your first 2 suggestions.

As for the last one, am I right in thinking this is about Atkins and similar (paleo/keto)? The nearest I have is [Protein Rich Foods and Gout](https://www.goutpal.com/1037/protein-rich-foods-current-recommendations/). But again, I need to rewrite that.

I'll get on with checking the latest studies. Along the way, I'll probably have to split this into several pages. In fact, that's what my original tips page should have been - a list of tips, with each tip fully explained on a separate page.

To that end, I'd like to ask every gout sufferer...
**What is your most effective gout diet tip, and how has it helped you control uric acid?**

**Comment by hfat** [comment]
Keto is a legitimate (though potentially dangerous) diet and doesn't necessarily involve overeating proteins but yeah, Atkins is probably the best-known diet which might lead people to grossly overeat proteins. More generally, I think most people would naturally tend to overeat proteins when maintaining weight loss so I didn't have a specific diet in mind actually. I probably overeat proteins by the way, but I try not to overeat too much (most days anyway). 
So while I think the page you link to about proteins is generally sound in its analysis of bizarre professional recommendations, it includes the unfortunate statement "you want to move towards increasing protein". People who are already overeating proteins should obviously not eat even more! And while excess protein intake should be matched by increased uric acid excretion in healthy individuals, one shouldn't assume uric acid excretion performs properly when someone has unexplained excess uric acid in their blood.
 
About tests and doctors, maybe I should have spent a few more seconds typing. I only meant professionals are useless when it comes to low-purine diets specifically and I was also thinking about tests related to potentially harmful diets and conditions requiring special diets. I wouldn't expect most doctors to bother with for uric excretion tests nowadays. My doctor keeps running useless tests because I don't eat corpses for instance. But these tests are so cheap compared to the consequences careless dietary restrictions might have that I don't mind. Doctors don't have time to learn who is and who is not actually at risk so let them test everyone who looks like they might possibly be at risk. 
Which brings me to... why would professionals want to learn about the effects of diet on uric acid? And who would want to do serious research about this nowadays? Drugs have become safe enough and work much better while there are so many patients afflicted by other conditions which make diet a life and death matter!
 
My most effective diet tip would be: if you have special dietary needs, don't eat socially (or rather, make that a rare occasion). This might seem obvious but we are social animals and it's easy to look the other way when it comes to the harm social eating causes to people with special needs.

**Comment by kct2020** [comment]
Thanks for your input - this is exactly what I need. And I will go through each point. But I started with "social eating" and I have some problems with explaining the issues.

Mainly because social eating studies seem mostly concerned with mental health. Especially, the benefits of eating socially in that context.

However, there is a substantial amount of work on "eating out", "fast-food" and similar aspects of eating food not cooked at home. Is that what you mean by "social eating"? Or have I missed the point?

**Comment by kct2020** [comment]
> while excess protein intake should be matched by increased uric acid excretion in healthy individuals, one shouldn't assume uric acid excretion performs properly when someone has unexplained excess uric acid in their blood.

That's a great way to explain the importance of testing diet changes in the same way an allopurinol patient would test dosing changes. So I'll definitely correct my unfortunate choice of words.

**Comment by kct2020** [comment]
> why would professionals want to learn about the effects of diet on uric acid? And who would want to do serious research about this nowadays? Drugs have become safe enough and work much better while there are so many patients afflicted by other conditions which make diet a life and death matter!

Again, well said!

I tend to sidestep this by relying on statements like "discuss the effects of your diet choices with your doctor". Knowing that most doctors will not be interested. But I think I need to consider making my suggestions more robust. For example,
Tip 1: Don't waste your time making diet changes that make no difference. If you can't show significant uric acid reduction after 1 month then start prescribed uric acid drugs to maintain safe levels.

I have an idea to add that it is possible to look at diet changes later. Once uric acid is safe. But I think that makes it weaker. Also, I can't recall a single forum discussion where anyone has tried that.

**Comment by hfat** [comment]
Yes, it's always a good idea to tell what you're doing to your doctor who may or may not order routine tests as a result. Just don't expect useful (or even factually-grounded) advice from them. 
But your "1 month" recommendation is only something I'd say to someone who obviously needs drugs urgently. It's a recipe for failure. If there's a realistic chance that a gout sufferer might manage their uric acid with diet, they'll need more time. Not only does changing one's diet often require substantially longer, we can only guess how low they need to go and their numbers are likely to be very noisy to begin with due to precipitation and dissolution so that a long time (possibly years) might be required for a clear picture to emerge. Certainly it is something that has often come up in the forum but I don't know if anyone ended up actually doing the tiresome work. Typically, diet is only a stage before people move on to drugs as people realize a daily pill isn't so bad once they get used to it but some people won't be able to because of the side effects and they do not need a recipe for failure designed to make other accept drugs! There is also the matter of people who should be reducing their dose due to side effects, and diet can help with that (granted, the incompetence of doctors means people may be told to quit the drug rather than lower the dose or alternatively they may not even be aware of the side effects).
 
As to social eating, while that obviously includes eating out, that would also include eating at home with one's family and whatnot. The issue is that it's all too easy (if you're not some kind of tyrant anyway) to eat according to other people's habits and needs. This isn't only about the foods but about when to eat and how much... and of course about reducing stress. As usual, there are other concerns such as income-related constraints or the mental health issue you brought up but people's behaviors can also be detrimental to one's mental health. And you're not going to be able to explain why people harm others (especially family members) on a website about gout!

**Comment by kct2020** [comment]
> As to social eating, while that obviously includes eating out, that would also include eating at home with one's family and whatnot

I've been thinking a lot about this. Though I'm not best placed to fully understand the pressures, as I mostly cook for just myself and eat alone. But I can definitely collect experiences of other gout sufferers and share them as a form of encouragement.

Today, I realized the scale of the problem when I read an article that isn't gout related. But it definitely brought home to me the scale of the problem.
<q cite="https://doi.org/10.1016/S1499-4046(06)60232-7">Family members had a substantial impact on women’s ability to make desired dietary changes. In some cases, women were encouraged by husbands or partners, children, parents, and/or siblings. For example, one participant, who had made extreme dietary changes in response to her cancer diagnosis, reported strong family support. Her parents shared their ideas, literature, and experiences concerning nutritional supplements and “health foods.” Her body-building sons were enthusiastic when she introduced vitamin and mineral supplements to the family. She described her husband as sometimes more enthused than she was [...]
About one third of the women,however,struggled against a certain amount of resistance from family members to diet changes that they wanted to make. </q>
`Beagan, Brenda L., and Gwen E. Chapman. "Eating after breast cancer: Influences on women's actions." Journal of nutrition education and behavior 36, no. 4 (2004): 181-188.`

The study goes on to list reasons for difficulties with post-diagnosis diet changes:

- the degree of family support or resistance
- employment situation
- resources
- cultural traditions
- concurrent health concerns.

All of which will affect any gout sufferers who want to manage their diet more effectively.

Anyway, I'll finish with my biggest tip for anyone considering a gout diet:
Start allopurinol or other uric acid lowering treatment first. Because that gives you your best chance of dissolving old crystal deposits that are spread throughout your body. This is particularly important because, if your diet changes are effective, they should cause that dissolving process to start. But that in turn causes fluctuations in blood test results that can significantly alter your perception of the benefits of foods that are good for uric acid. So wait until the "debulking phase" of uric acid treatment is complete. Then use dietary changes to reduce or remove the need for drugs.

**Comment by hfat** [comment]
Sorry to keep harping on this but some people can't take the drugs that actually work (or simply can't get a doctor who knows enough to put them on the right drug). 
So I think there's a difficult balance to strike : discourage the majority who are reticent to take the drugs from focusing on their diet while encouraging the few who actually need to tailor their diet to their unusual needs to put in the considerable effort required in spite of the discouraging results they are likely to experience after heeding bad dietary advice. 
Some people might also need to change their diet in order to be able to take uric acid drugs (or, once again, their doctors might simply assume this is the case because they don't have a clue about drugs). But if the issue is side-effects which might be mitigated by dietary changes, the focus should of course not be on purines. 
Add to this cultural issues and indeed concurrent health concerns, and it's impossible to recommend a gout diet (or a side-effect diet) suitable to most. In any case, the lack of good evidence for the effectiveness of dietary changes is going to make the whole process especially challenging for social eaters. I mostly cook for myself and eat alone as well, but I used not to and I only realized after the fact how much easier it would have made experimenting with potential solutions. The catch is that part of the reason I used not to cook for myself as often was frequently debilitating gout-like symptoms... no doubt caused by doctors not giving me uric acid drugs (and perhaps by doctors not giving me the right drug combination to deal with inflammation as well).
